src/System.IO/src/System/IO/TextReader.cs

```diff
@@ -265,5 +266,123 @@ public override string ReadLine()
                 return null;
             }
         }
+
+#if netstandard17
```

Comment:
Instead of having all of this code here, could you please turn this file into a partial class and then have a separate file that has the netstandard1.7 code and we only add it when TargetGroup is empty?